Microsoft.O365.Security.Native.ETW/IEventRecord.hpp

```diff
@@ -402,6 +402,12 @@ namespace Microsoft { namespace O365 { namespace Security { namespace ETW {
         /// <param name="result">the resulting byte array</param>
         /// <returns>true if fetching the data succeeded, false otherwise</returns>
         bool TryGetBinary(String^ name, [Out] array<Byte>^% result);
+
+        /// <summary>
+        ///  Retrieves the call stack associated with the record, if enabled.
+        /// </summary>
+        /// <returns>a list of return addresses</returns>
+        List<UIntPtr>^ StackTrace();
```

Comment:
optional: I think I'd prefer `GetStackTrace()` here since this is a method rather than a property.

Reply:
Would you prefer the C++ version be renamed to `get_stack_trace()` also?

Also, I currently wrap the stack trace in a `std::vector` - but I could instead return a pointer to the data in the record and let the caller copy if required? Or perhaps even expose both options?

Reply:
The C++ naming seems idiomatic as-is, so let's leave it. Since `EVENT_EXTENDED_ITEM_STACK_TRACE64` seems to only be a list of addresses I think a `std::vector` of them is a good approach.